provider/pkg/await/await.go

```diff
@@ -691,3 +692,14 @@ func checkIfResourceDeleted(
 func clearStatus(context context.Context, host *pulumiprovider.HostClient, urn resource.URN) error {
 	return host.LogStatus(context, diag.Info, urn, "")
 }
+
+// patchForce decides whether to overwrite patch conflicts.
+func patchForce(obj *unstructured.Unstructured) bool {
+	if metadata.IsAnnotationTrue(obj, metadata.AnnotationPatchForce) {
+		return true
+	}
+	if enabled, exists := os.LookupEnv("PULUMI_K8S_ENABLE_PATCH_FORCE"); exists {
+		return enabled == "true"
```

Comment:
Nit: Might be nice if the value could be any case or 1 (i.e. "truthy"), but I wouldn't block on it, this is fine.

Reply:
Yep, I followed precedence for other env vars on this provider, but it would be a nice thing to change. I'll open an issue to follow up on that.